spec/requests/api/v1/moves_controller_update_spec.rb

```diff
@@ -113,11 +113,11 @@
         end
 
         it 'updates the cancellation_reason of a move' do
```

Comment:
can we change the description to `does not update`?

Reply:
now fixed